docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,199 @@
+---
+id: dual-region
+title: "Dual-Region"
+sidebar_label: "Dual-Region"
+description: "Dual-Region allows you to run Camunda Platform connected in two regions."
+---
+
+import DualRegion from "./img/dual-region.svg";
+
+## Introduction
+
+Camunda 8 is compatible with a dual-region setup under certain [limitations](#limitations). This allows Camunda 8 to run in a mix of active-active and active-passive setups, resulting in an overall **active-passive** setup. The following will explore the concept, limitations, and considerations.
+
+:::warning
+
+You should get familiar with the topic, the [limitations](#limitations) of the dual-region setup, and the general [considerations](#considerations) on operating a dual-region setup.
+
+:::
+
+## Disclaimer
+
+:::danger
+
+- Customers must develop and test [operational procedures](<!-- TODO: Link -->) in non-production environments based on the framework steps outlined by Camunda, **before applying them in production setups**.
+- Before advancing to production go-live, customers need to validate these procedures with Camunda.
+- Customers are solely responsible for detecting any regional failures and implementing the necessary [operational procedures](<!-- TODO: Link -->).
+
+:::
+
+## Architecture
+
+<DualRegion />
+
+The illustrated architecture consists of two regions. Each region houses a Kubernetes cluster in which Camunda 8 is deployed. Those two Kubernetes clusters are capable of communicating with each other.
+
+One of the regions will be considered **active** and the other one **passive**. User traffic must only reach the **active** region. We consider **Region 0**, the one underlined in green, to be the active region and **Region 1** to be the passive region. In our case, user traffic would only go to **Region 0**. **Region 1** would be considered passive and used in case of the loss of the active region. Due to Zeebe's data replication, you can recover from an active region loss by utilizing the passive region without much downtime.
+
+Zeebe stretches across the regions due to its use of the [Raft protocol](<https://en.wikipedia.org/wiki/Raft_(algorithm)>) allowing it to communicate and replicate data between all brokers across. Zeebe is exporting data to two Elasticsearch instances, one in each region. Operate and Tasklist will import the previously exported data and run per region.
+
+The currently supported Camunda 8 components are the following:
+
+- Zeebe (Workflow Engine)
```

Comment:
Should likely mention the others as well, such as Connectors, Optimize, Identity, etc.